azure/azure_test.go

```diff
@@ -83,14 +84,23 @@ func Test_Examples(t *testing.T) {
 				"@pulumi/cloud-azure",
 			},
 		},
-		{
-			Dir:    path.Join(cwd, "./examples/cloud-ts-thumbnailer"),
-			Config: commonConfig,
-			Dependencies: []string{
-				"@pulumi/cloud",
-				"@pulumi/cloud-azure",
-			},
-		},
+		// TODO(cyrusn): These are currently timing out on travis.
+		// {
+		// 	Dir:    path.Join(cwd, "./examples/cloud-ts-thumbnailer"),
+		// 	Config: commonConfig,
+		// 	Dependencies: []string{
+		// 		"@pulumi/cloud",
+		// 		"@pulumi/cloud-azure",
+		// 	},
+		// },
+		// {
+		// 	Dir:    path.Join(cwd, "./examples/containers"),
+		// 	Config: commonConfig,
+		// 	Dependencies: []string{
+		// 		"@pulumi/cloud",
+		// 		"@pulumi/cloud-azure",
+		// 	},
+		// },
```

Comment:
Does this mean we won't have test coverage on the new functionality being added here?

I'd feel much better about adding all this new code into the framework if we had some tests covering it.

Reply:
Not intentional.  Just so i could temporarily test things like travis was having some trouble talking ot Azure.